### PR TITLE
Add wip column to CustomAssessments

### PR DESCRIPTION
**Pull request by gigxz**

### CODE REVIEW

File: drivers/hmis/app/jobs/hmis/merge_clients_job.rb [hunk 1]
```diff
@@ -135,6 +135,7 @@ def update_client_id_foreign_keys
         GrdaWarehouse::ClientFile,
         Hmis::File,
         Hmis::Wip,
+        HmisExternalApis::AcHmis::ReferralHouseholdMember,
```
Comment (gigxz) [inline drivers/hmis/app/jobs/hmis/merge_clients_job.rb:138]: Unrelated change: update `client_id` on this table when clients are merged

**Comment by gigxz** [comment]
deployed and QA'd on QA

**Review by eanders — APPROVED** [review]
This looks _way_ cleaner to me.  Thanks!
